modules/podman_image_v2.py

```diff
@@ -15,23 +15,25 @@
       - Renu Hadke (@renuHadke)
       - Neha Jain (@nehajain1809)
       - Krish Jain (@krishjain4894)
-      - Chinmay Keskar (@ckeskar)
```

Comment:
Why is this name removed within this commit?

Frankly:  I think it is by accident.